searchapp/app/routes/developments/search.js

```diff
@@ -27,17 +28,18 @@ export default Ember.Route.extend({
     queryObject.page["number"] = params["number"]
     queryObject.page["size"] = params["size"]
 
-
-
-    return this.store.find("development", queryObject);
+    return this.store.query("development", queryObject).then(function(model) {
+      console.log(App);
+      model.meta = App.storeMeta['development'];
```

Comment:
I don't usually care about Hound's JS notices, but if it's correct about this dot notation thing, I agree.